### ISSUE TITLE
Can't add any new Philips led

### ISSUE THREAD

**Issue by Kheamu** [Whilser/Philips-Zhirui-LED-lamp#6]
Hello! Thx for you work!
I added one philips device and it's work good. But with another there is such mistake: Error: (PhilipsLED) failed to load 'plugin.py' . I am noob in programming. What can i do? Sorry for my english.

**Comment by Kheamu** [comment]
Full text of mistake:

 2019-09-17 18:22:15.966 Error: (PhilipsLED) failed to load 'plugin.py', Python Path used was '/home/pi/domoticz/plugins/PhilipsLED/:/usr/lib/python37.zip:/usr/lib/python3.7:/usr/lib/python3.7/lib-dynload:/usr/local/lib/python3.7/dist-packages:/usr/lib/python3/dist-packages:/usr/lib/python3.7/dist-packages'.

**Comment by Kheamu** [comment]
Reinstalled python-miio

2019-09-17 19:09:14.455 Error: (PhilipsLED) failed to load 'plugin.py', Python Path used was '/home/pi/domoticz/plugins/PhilipsLED/:/usr/lib/python37.zip:/usr/lib/python3.7:/usr/lib/python3.7/lib-dynload:/usr/local/lib/python3.7/dist-packages:/usr/lib/python3/dist-packages:/usr/lib/python3.7/dist-packages'.
2019-09-17 19:09:14.456 Error: (BL) Module Import failed, exception: 'RuntimeError'
2019-09-17 19:09:14.456 Error: (BL) Error Line details not available.
2019-09-17 19:10:15.962 Error: BL hardware (5) thread seems to have ended unexpectedly
2019-09-17 19:10:29.965 Error: BL hardware (5) thread seems to have ended unexpectedly
2019-09-17 19:10:43.967 Error: BL hardware (5) thread seems to have ended unexpectedly

PS BL - my lamp.

**Comment by Whilser** [comment]
Hi! The plugin tested on raspbian stretch and python 3.5 installed and works fine. Raspbian buster is not yet tested.

**Comment by Kheamu** [comment]
Tell me please how can i install phyton 3.5 in such case?

**Comment by Whilser** [comment]
Try to use raspbian stretch for now https://downloads.raspberrypi.org/raspbian/images/raspbian-2019-04-09/

**Comment by Kheamu** [comment]
Thank you very much for answers! I will try it and then write here.

**Comment by Kheamu** [comment]
All work on raspbian stretch system. Added all my 8 xiaomi philips devices. Thanks a lot!

**Comment by Tominjoo** [comment]
@Whilser Hi Whisler, thx for the great add-on. 

I have the same issue. However i already succeeded on installing 1 of my lights. but get the same error as Kheamu on my second light. I am running Buster. but want to ask before reinstalling my domoticz :) 
Do i need to switch to stretch as well? or can it be light or something? 

(PhilipsLED) failed to load 'plugin.py', Python Path used was '/home/pi/domoticz/plugins/PhilipsLED/:/usr/lib/python37.zip:/usr/lib/python3.7:/usr/lib/python3.7/lib-dynload:/usr/local/lib/python3.7/dist-packages:/usr/lib/python3/dist-packages:/usr/lib/python3.7/dist-packages'.
2019-10-01 17:28:58.653 Error: (groene lamp) Module Import failed, exception: 'RuntimeError'
2019-10-01 17:28:58.653 Error: (groene lamp) Error Line details not available. 

this is the error. 
Thx in advance!

**Comment by Whilser** [comment]
Yes, on raspbian stretch with python 3.5 everything works fine. Buster with python 3.7 have not been tested with the plugin.

**Comment by Tominjoo** [comment]
@Whilser Thank you for this quick response! i'll be doing that soon then! thank you

**Comment by mazycz** [comment]
I've made a fork of this plugin and solve issue with multiple lights on Raspbian Buster and Python 3.7.
It is still not done, but its can be already used. Big difference is in specifiing all devices in one hardware.
https://github.com/mazycz/Xiaomi-MiJia-Philips-LED-Lamp

Currently I'm using it with 11 lights, without scenes.
